core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -21,6 +21,7 @@
 #include <impl/Kokkos_Tools.hpp>
 #include <atomic>
 #include <Cuda/Kokkos_Cuda_Error.hpp>
+#include <cuda_runtime_api.h>
```

Comment:
I guess you remove this include from all files that include this file?

Reply:
Yes, since there are no runtime api calls anywhere else now. But maybe this file includes types that are still used in the other `core/src/Cuda/` files?

Reply:
I have a branch that adds the test that checks whether or not header files are self contained for the `core/src/Cuda` files. This would catch that case I believe?

Reply:
I think it's fine.